src/main/groovy/com/ullink/NuGetPack.groovy

```diff
@@ -69,16 +70,8 @@ class NuGetPack extends BaseNuGet {
         super.exec()
     }
 
-    void nuspec(Closure closure) {
```

Comment:
any chance we can keep the backward compat by keeping this method and forwarding the setup to nugetSpec task ?